ci/cloudbuild/build.sh

```diff
@@ -112,7 +112,7 @@ if [ "${DISTRO}" != "local" ]; then
   exec gcloud builds submit "${args[@]}" .
 fi
 
-io::log "====> STARTING BUILD: ${BUILD_NAME}"
+io::log_h1 "STARTING BUILD: ${BUILD_NAME}"
```

Comment:
Perhaps you don't need to yell "STARTING BUILD" quite so loudly now?

Reply:
FINE! ;-)